ArgosIOS/Argos/Views/LandingPage/LandingViewModel.swift

```diff
@@ -55,6 +58,17 @@ class LandingViewModel: LoadableObject {
                         }
                     }
                     .store(in: &self.cancellables)
+                self.socketClient.$coords
+                    .sink { [weak self] coords in
```

Comment:
so this is good, but you should make a viewmodel for the map view and put this there rather than in here